codecov.report

```diff
@@ -0,0 +1,14 @@
+ok  	istio.io/pilot/adapter/config/aggregate	0.291s	coverage: 92.6% of statements
```

Comment:
I don't think this should be checked in ?

Reply:
Thanks!